WooCommerce/Classes/POS/Presentation/Settings/PointOfSaleSettingsView.swift

```diff
@@ -0,0 +1,223 @@
+                        selection = .help
+                    } label: {
+                        HStack {
+                            Image(systemName: SidebarNavigation.help.icon)
+                                .font(.posBodyLargeRegular())
+                                .foregroundStyle(selection == .help ? .white : .primary)
+                            VStack(alignment: .leading) {
+                                Text(SidebarNavigation.help.title)
+                                    .font(.posBodyLargeRegular())
+                                    .foregroundStyle(selection == .help ? .white : .primary)
+                                Text(SidebarNavigation.help.subtitle)
+                                    .font(.posBodyMediumRegular())
+                                    .foregroundStyle(selection == .help ? .secondary : .secondary)
+                            }
+                            Spacer()
+                        }
+                        .padding(.vertical, POSPadding.small)
+                        .padding(.horizontal, POSPadding.medium)
+                        .contentShape(Rectangle())
+                    }
+                    .buttonStyle(.plain)
+                    .background(
+                        RoundedRectangle(cornerRadius: POSCornerRadiusStyle.large.value, style: .continuous)
+                            .fill(selection == .help ? Color.accentColor : Color.clear)
+                    )
+                }
+            }
+            Group {
+                switch selection {
+                case .store:
+                    PointOfSaleSettingsStoreDetailView()
+                case .hardware:
+                    PointOfSaleSettingsHardwareDetailView()
+                case .help:
+                    PointOfSaleSettingsHelpDetailView()
+                default:
+                    EmptyView()
+                }
+            }
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
+        }
+    }
+}
+
+extension PointOfSaleSettingsView {
+    enum HardwareDestination: Identifiable, CaseIterable {
+        case cardReaders
+        case scanners
+
+        var id: Self { self }
+
+        var title: String {
+            switch self {
+            case .cardReaders:
+                return Localization.hardwareNavigationCardReaderTitle
+            case .scanners:
+                return Localization.hardwareNavigationBarcodeTitle
+            }
+        }
+
+        var subtitle: String {
+            switch self {
+            case .cardReaders:
+                return Localization.hardwareNavigationCardReaderSubtitle
+            case .scanners:
+                return Localization.hardwareNavigationBarcodeSubtitle
+            }
+        }
+
+        var icon: String {
+            switch self {
+            case .cardReaders:
+                return "creditcard"
+            case .scanners:
+                return "qrcode.viewfinder"
+            }
+        }
+    }
+}
```

Comment:
It feels like overreach for this to be defined in the settings view. Surely this would be better in `PointOfSaleSettingsHardwareDetailView`?

Reply:
True that, I'll address this one along https://github.com/woocommerce/woocommerce-ios/pull/16038 , as `PointOfSaleSettingsView.HardwareDestination` changes 👍